### PR TITLE
added steps to verify that dependencies are installed

### PR DESCRIPTION
**Pull request by ericsysmin**

### CODE REVIEW

File: tasks/python_requirements.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+- name: AviSDK | Requirements | python pip package is installed
+  command: which pip
+  changed_when: false
```
Comment (grastogi23) [inline tasks/python_requirements.yml]: we don't need to make python pip as required. Some installations have used direct install as well. in fact in our controller and jenkins sdk is not installed via pip.

File: tasks/python_requirements.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+- name: AviSDK | Requirements | avisdk python package is installed
+  command: pip show avisdk
```
Comment (grastogi23) [inline tasks/python_requirements.yml]: also don't modules give the error that avisdk is not installed.

Comment (ericsysmin) [reply tasks/python_requirements.yml]: got a better way to check?

File: tasks/python_requirements.yml (hunk 1 above)
Comment (ericsysmin) [reply tasks/python_requirements.yml]: Yea, I figured that, is there a better way to check if a library exists without using pip?

File: tasks/python_requirements.yml (hunk 2 above)
Comment (grastogi23) [reply tasks/python_requirements.yml]: https://github.com/avinetworks/ansible-role-avisdk/blob/e05d06f488a07a581bda8a76b76b35286f7d03bb/library/avi_pool.py#L372

File: tasks/python_requirements.yml (hunk 1 above)
Comment (grastogi23) [reply tasks/python_requirements.yml]: no I could not find a consistent way to solve this so ended up coding it as part of module. That is also inline with ansible core guidelines.
https://github.com/avinetworks/ansible-role-avisdk/blob/e05d06f488a07a581bda8a76b76b35286f7d03bb/library/avi_pool.py#L372

**Pull request closed without merging**